Apps/GatewayApi/src/Models/PatientDetails.cs

```diff
@@ -0,0 +1,84 @@
+// -------------------------------------------------------------------------
+//  Copyright © 2019 Province of British Columbia
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+// -------------------------------------------------------------------------
+namespace HealthGateway.GatewayApi.Models
+{
+    using System;
+    using System.Text.Json.Serialization;
+    using HealthGateway.AccountDataAccess.Patient;
+
+    /// <summary>
+    /// The patient details.
+    /// </summary>
+    public class PatientDetails
+    {
+        /// <summary>
+        /// Gets or sets the health directed identifier.
+        /// </summary>
+        [JsonPropertyName("hdid")]
+        public string HdId { get; set; } = string.Empty;
+
+        /// <summary>
+        /// Gets or sets the patient's PHN.
+        /// </summary>
+        [JsonPropertyName("personalHealthNumber")]
+        public string Phn { get; set; } = string.Empty;
+
+        /// <summary>
+        /// Gets or sets the patient's common name.
+        /// </summary>
+        [JsonPropertyName("commonName")]
```

Comment:
These can be left out when the provided value matches the camel-cased version of the property name, since that is the default value.  Applies to all the properties in this file except hdid and personalHealthNumber.